hack/deploy-logging.sh

```diff
@@ -150,75 +236,67 @@ wait_for_logging_is_running() {
 }
 
 deploy_logging_using_olm() {
-    # Create the $ESO_NS namespace:
-    if oc get project $ESO_NS > /dev/null 2>&1 ; then
-        echo using existing project $ESO_NS
-    else
-        oc create -f $TEST_OBJ_DIR/openshift-operators-redhat-namespace.yaml
-    fi
-    oc get projects | grep logging || :
-
-    # Create an OperatorGroup for $LOGGING_NS:
-    oc -n ${LOGGING_NS} create -f $TEST_OBJ_DIR/openshift-logging-operatorgroup.yaml
-    # Create an OperatorGroup for $ESO_NS:
-    oc -n $ESO_NS create -f $TEST_OBJ_DIR/openshift-operators-redhat-operatorgroup.yaml
-
-    # Create the CatalogSourceConfig for the elasticsearch-operator in the namespace openshift-marketplace:
-    oc create -n openshift-marketplace -f $TEST_OBJ_DIR/elasticsearch-catalogsourceconfig.yaml
-    oc get -n openshift-marketplace CatalogSourceConfig | grep elasticsearch || :
-
-    # Create the subscription for elasticsearch in the namespace $ESO_NS:
-    oc create -n $ESO_NS -f $TEST_OBJ_DIR/elasticsearch-subscription.yaml
-    oc get -n $ESO_NS subscriptions | grep elasticsearch || :
-
-    # Create the CatalogSourceConfig for cluster-logging in the namespace openshift-marketplace:
-    oc create -n openshift-marketplace -f $TEST_OBJ_DIR/cluster-logging-catalogsourceconfig.yaml
-    oc get -n openshift-marketplace CatalogSourceConfig | grep logging || :
-
-    # create the subscription in the namespace $LOGGING_NS:
-    oc create -n ${LOGGING_NS} -f $TEST_OBJ_DIR/cluster-logging-subscription.yaml
-    oc get -n ${LOGGING_NS} subscriptions | grep logging || :
+    local manifest=$( mktemp -d )
+    trap "rm -rf $manifest" RETURN
+
+    local eoimg=${EO_IMAGE:-$( construct_image_name elasticsearch-operator latest )}
+    cp -r ${EO_DIR}/manifests/${MASTER_VERSION} $manifest
```

Comment:
Not sure if its worth modifying here but I have a utility I was working on that pulls the manifest from an image with something like:
```
         container=$($IMAGE_TOOL create $(echo $IMAGE | cut -c1-5 --complement))
         $IMAGE_TOOL cp $container:/manifests ${dir}
```

Reply:
ok - I'll investigate that

Reply:
I did not try this in CI